cloudslang-runtime/src/main/resources/scripts/cs_extract_number.py

```diff
@@ -0,0 +1,12 @@
+def cs_extract_number(str, count = 1):
+  nums = str.split()
```

Comment:
@alexbajzat if the variable is not a string no need to force it, it is ok to fail, since the user choose this. Previously the meaning of the function was changed.